R/calcEDGETrData.R

```diff
@@ -17,18 +17,16 @@ calcEDGETrData <- function() {
     c("SSP1", "SSP2", "SSP5", "SSP2EU", "SDP"),
     c("ConvCase", "ElecEra", "HydrHype", "Mix",
       "Mix1", "Mix2", "Mix3", "Mix4")), split=".", fixed=TRUE)
-
-  ## add both smartlifestyle and default lifestyle variants for all scenarios
-  allscens <- append(
-    lapply(edgetScenarios, function(sc){c(sc, TRUE)}),
-    lapply(edgetScenarios, function(sc){c(sc, FALSE)}))
   ## SHAPE scenarios are coupled to specific technologies
   allscens <- append(
     allscens,
     list(
       c("SDP_EI", "ElecEra", FALSE),
-      c("SDP_MC", "ElecEra", TRUE),
-      c("SDP_RC", "ElecEra", TRUE)))
+      c("SDP_MC", "ElecEra", FALSE),
+      c("SDP_RC", "ElecEra", FALSE)))
+
+  ## add default lifestyle variants for all scenarios (exclude lifestyle)
+  allscens <- lapply(allscens, function(sc){c(sc, FALSE)})
```

Comment:
this is a strange way to do it. I would just not add this parameter to the list and set it to FALSE in the call.